test/dataframerow.jl

```diff
@@ -42,27 +47,41 @@ end
 
     r = DataFrameRow(df, 2, :)
     @test r[:] === r
+    @test isequal(r[r""], r)
     @test view(r, :) === r
+    @test isequal(view(r, r""), r)
     @test r[3] == "B"
+    @test r[r"c"] == r[[3]]
     @test_throws BoundsError r[5]
     @test view(r, 3)[] == "B"
+    @test view(r, r"c")[1] == "B"
+    view(r, r"c") == view(r, [3])
     view(r, 3)[] = "BB"
     @test df.c[2] == "BB"
     @test_throws MethodError r[true]
     @test_throws MethodError view(r, true)
     @test copy(r[[:c,:a]]) == (c = "BB", a = 2)
+    @test copy(r[r"[ac]"]) == (a = 2, c = "BB")
+    @test copy(r[r"x"]) == NamedTuple()
```

Comment:
Should this also be tested for an empty vector of symbols? Same below.

Reply:
fixed